webapp-django/crashstats/authentication/management/commands/migrate-users-csv.py

```diff
@@ -1,42 +0,0 @@
-import csv
```

Comment:
We don't need this anymore--it's for migration users from their email alias to their LDAP account.

Reply:
I guess this is because of the recent move to Mozilla Single Sign On?

Reply:
This is leftover from the previous auth migration when we moved from Persona to Google Auth.